2024-11-06 16:15:02.086 [info] (Client) Pylance async client (2024.11.1) started with python extension (2024.18.0)
2024-11-06 16:15:03.007 [info] [Info  - 4:15:03 PM] (1742) Server root directory: file:///home/felipersas/.vscode/extensions/ms-python.vscode-pylance-2024.11.1/dist
2024-11-06 16:15:03.066 [info] [Info  - 4:15:03 PM] (1742) Pylance language server 2024.11.1 (pyright version 1.1.386, commit 4180fa21) starting
2024-11-06 16:15:03.080 [info] [Info  - 4:15:03 PM] (1742) Starting service instance "Problemas-Classicos-de-CC---Livro"
2024-11-06 16:15:03.351 [info] [Info  - 4:15:03 PM] (1742) Setting environmentName for service "Problemas-Classicos-de-CC---Livro": "3.12.7 (global)"
2024-11-06 16:15:03.352 [info] [Info  - 4:15:03 PM] (1742) Setting pythonPath for service "Problemas-Classicos-de-CC---Livro": "/bin/python"
2024-11-06 16:15:03.353 [info] [Info  - 4:15:03 PM] (1742) No include entries specified; assuming /home/felipersas/Documents/Problemas-Classicos-de-CC---Livro
2024-11-06 16:15:03.354 [info] [Info  - 4:15:03 PM] (1742) Auto-excluding **/node_modules
2024-11-06 16:15:03.354 [info] [Info  - 4:15:03 PM] (1742) Auto-excluding **/__pycache__
2024-11-06 16:15:03.355 [info] [Info  - 4:15:03 PM] (1742) Auto-excluding **/.*
2024-11-06 16:15:03.429 [info] [Info  - 4:15:03 PM] (1742) Assuming Python version 3.12.7.final.0
2024-11-06 16:15:03.605 [info] [Info  - 4:15:03 PM] (1742) Found 12 source files
2024-11-06 16:15:04.860 [info] [Info  - 4:15:04 PM] (1742) Background analysis(1) root directory: file:///home/felipersas/.vscode/extensions/ms-python.vscode-pylance-2024.11.1/dist
2024-11-06 16:15:04.866 [info] [Info  - 4:15:04 PM] (1742) Background analysis(1) started
2024-11-06 16:15:05.056 [info] [Info  - 4:15:05 PM] (1742) Indexer background runner(2) root directory: file:///home/felipersas/.vscode/extensions/ms-python.vscode-pylance-2024.11.1/dist (index)
2024-11-06 16:15:05.056 [info] [Info  - 4:15:05 PM] (1742) Indexing(2) started
2024-11-06 16:15:05.961 [info] [Info  - 4:15:05 PM] (1742) scanned(2) 15 files over 1 exec env
2024-11-06 16:15:06.326 [info] [Info  - 4:15:06 PM] (1742) indexed(2) 15 files over 1 exec env
2024-11-06 16:15:06.359 [info] [Info  - 4:15:06 PM] (1742) Indexing finished(2).
2024-11-06 16:34:05.983 [info] [Info  - 4:34:05 PM] (1742) Starting service instance "<default>"
2024-11-06 16:34:06.095 [info] [Info  - 4:34:06 PM] (1742) Setting environmentName for service "<default>": "3.12.7 (global)"
2024-11-06 16:34:06.095 [info] [Info  - 4:34:06 PM] (1742) Setting pythonPath for service "<default>": "/bin/python"
2024-11-06 16:34:06.096 [info] [Info  - 4:34:06 PM] (1742) No include entries specified; assuming /<default workspace root>
2024-11-06 16:34:06.096 [info] [Info  - 4:34:06 PM] (1742) Auto-excluding **/node_modules
2024-11-06 16:34:06.096 [info] [Info  - 4:34:06 PM] (1742) Auto-excluding **/__pycache__
2024-11-06 16:34:06.097 [info] [Info  - 4:34:06 PM] (1742) Auto-excluding **/.*
2024-11-06 16:34:06.160 [info] [Info  - 4:34:06 PM] (1742) Assuming Python version 3.12.7.final.0
2024-11-06 16:34:06.211 [info] [Error - 4:34:06 PM] (1742) File or directory "/<default workspace root>" does not exist.
2024-11-06 16:34:06.211 [info] [Info  - 4:34:06 PM] (1742) No source files found.
2024-11-06 16:34:06.805 [info] [Info  - 4:34:06 PM] (1742) Background analysis(3) root directory: file:///home/felipersas/.vscode/extensions/ms-python.vscode-pylance-2024.11.1/dist
2024-11-06 16:34:06.808 [info] [Info  - 4:34:06 PM] (1742) Background analysis(3) started
2024-11-06 17:24:51.397 [info] [Info  - 5:24:51 PM] (1742) Found 12 source files
